README.md

```diff
@@ -1,10 +1,106 @@
-OpenCensus - A stats collection and distributed tracing framework
-=================================================================
+# OpenCensus - A stats collection and distributed tracing framework
 [![Build Status][travis-image]][travis-url] [![Build status][appveyor-image]][appveyor-url] [![Maven Central][maven-image]][maven-url]
 
-OpenCensus provides a framework to define and collect stats against metrics and to
-break those stats down across user-defined dimensions. The library is in alpha
-stage and the API is subject to change.
+OpenCensus provides a framework to define and collect stats against metrics and to break those 
+stats down across user-defined dimensions. The library is in alpha stage and the API is subject 
+to change.
+
+## Quickstart for Third Party Libraries
+
+In general third party libraries only need to record tracing/stats events and propagate the context.
+
+### Add the dependencies to your project
+
+For Maven add to your `pom.xml`:
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-api</artifactId>
+    <version>0.6.0</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle add to your dependencies:
+```gradle
+compile 'io.opencensus:opencensus-api:0.6.0'
+```
+
+### Hello "OpenCensus" trace events
+
```

Comment:
Here's an example of doing X. Notice Y is possible because we propagate scope. 3rd parties libraries like SLF4J can integrate the same way.

Reply:
done.